rtrlib/rtr/packets.c

```diff
@@ -1130,8 +1191,9 @@ int rtr_send_error_pdu(const struct rtr_socket *rtr_socket, const void *erroneou
 	err_pdu->len = msg_size;
 
 	err_pdu->len_enc_pdu = erroneous_pdu_len;
-	if (erroneous_pdu_len > 0)
+	if (erroneous_pdu_len > 0) {
```

Comment:
IIRC this braces are not allowed in kernel coding style:

> Do not unnecessarily use braces where a single statement will do.

As shown 5 lines below